relion/protocols/protocol_base.py

```diff
@@ -1131,7 +1122,7 @@ def _getIterClasses(self, it, clean=False):
         if clean:
             pwutils.cleanPath(data_classes)
 
-        if not os.path.exists(data_classes):
+        if not pwutils.exists(data_classes):
```

Comment:
This usage is misleading...the pwutils does not have an 'exists' method, it is somewhere imported from os.path, but this is implementation details, so it should not be used. It is better to use os.path.exists directly. Please check all occurrences of this.

Reply:
oh no, I've been replacing it for years everywhere...